src/kernels/types.ts

```diff
@@ -611,7 +611,7 @@ export const IKernelFinder = Symbol('IKernelFinder');
 
 export interface IKernelFinder {
     onDidChangeKernels: Event<void>;
-    listKernels(resource: Resource, cancelToken?: CancellationToken): Promise<KernelConnectionMetadata[]>;
+    listKernels(cancelToken?: CancellationToken): Promise<KernelConnectionMetadata[]>;
```

Comment:
In a subsequent PR this will change to be a property instead of an async method.